crates/scsi/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "vhost-device-scsi"
+version = "0.1.0"
+authors = ["Gaelan Steele <gbs@canishe.com>"]
+edition = "2018"
```

Comment:
Other crates have 2021 here.

Can you please cross check with any other crate, like gpio and see if there are any differences that we can get rid of ?